bentoml/saved_bundle/loader.py

```diff
@@ -158,6 +159,25 @@ def load_bento_service_class(bundle_path):
     return model_service_class
 
 
+def safe_retrieve(bundle_path, target_dir):
+    """Safely retrieve bento service to local path
+
+    Args:
+        bundle_path (str): The path that contains saved BentoService bundle,
+            supporting both local file path and s3 path
+        target_dir (str): Where the service contents should end up
+
+    Returns:
+        string: location of safe local path
+    """
+    if _is_remote_path(bundle_path):
+        with _resolve_remote_bundle_path(bundle_path) as local_bundle_path:
+            shutil.copytree(local_bundle_path, target_dir)
+            return target_dir
+
+    return bundle_path
```

Comment:
Shouldn't it also copy this local `bundle_path`  to the `target_dir`?

Reply:
Thats a dood point...I suppose it should, for consistency sake. 👍

Reply:
Now it will also copy this local bundle_path to the target_dir